lib/util.js

```diff
@@ -211,25 +211,37 @@ const uploadExtension = (name, id, version, crxFile) => {
 
         var params = {
           Bucket: S3_EXTENSIONS_BUCKET,
-          Key: `release/${id}/${previousComponentFilename}`,
-          Tagging: {
-            TagSet: [
-              {
-                Key: `${componentNameTag}`,
-                Value: `${getPreviousVersion(version).replace(/\./g, '.')}`
-              }
-            ]
-          }
+          Key: `release/${id}/${previousComponentFilename}`
         }
-        aws_s3.putObjectTagging(params, function(err, data) {
-          if (err) {
-            console.error(`Tagging failed for ${id}/${previousComponentFilename} with tag ${getPreviousVersion(version).replace(/\./g, '.')}`)
-            console.error(err, err.stack)
-            reject(new Error('Failed to upload extension to S3'))
+
+        aws_s3.headObject(params, function (err, metadata) {
+          if (err && err.code === 'NotFound') {
```

Comment:
With this new check, do we still need `if (version == FirstVersion) {` check above?

Reply:
Great catch! thanks @yrliou